sdk/storage/azure-storage-file-datalake/out/azure/storage/filedatalake/_file_system_client.pyi

```diff
@@ -27,5 +27,7 @@ class FileSystemClient(StorageAccountHostsMixin):
     def delete_directory(self, directory: Union[DirectoryProperties, str], **kwargs: Any) -> DataLakeDirectoryClient: ...
     def create_file(self, file: Union[FileProperties, str], **kwargs: Any) -> DataLakeFileClient: ...
     def delete_file(self, file: Union[FileProperties, str], lease: Optional[Union[DataLakeLeaseClient, str]]=..., **kwargs: Any) -> DataLakeFileClient: ...
+    def set_file_system_access_policy(self, signed_identifiers: Dict[str, AccessPolicy], public_access: Optional[Union[str, PublicAccess]]=..., **kwargs: Any) -> Dict[str, Union[str, datetime]]: ...
```

Comment:
How often do we expect people to provide a `public_access` parameter? What happens if I don't (does it clear the previous value or keep it?)

Reply:
It's supposed to set together with signed_identifiers, else public_access will be set to None by default.